.docker/Dockerfile

```diff
@@ -0,0 +1,115 @@
+# syntax = docker/dockerfile:1.3
+
+# ghcr.io/ros-planning/moveit2:${ROS_DISTRO}-source
+# Downloads the moveit source code and install remaining debian dependencies
+
+ARG ROS_DISTRO=rolling
+
+######################### Base Tutorial Image  #########################################
+
+FROM moveit/moveit2:${ROS_DISTRO}-source as base_image
+
+# Export ROS_UNDERLAY for downstream docker containers
+ENV ROS_UNDERLAY /root/ws_moveit/install
+WORKDIR $ROS_UNDERLAY/..
+
+# Copy MoveIt sources from docker context
+COPY . src/moveit2_tutorials
+
+# Commands are combined in single RUN statement with "apt/lists" folder removal to reduce image size
+# https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers
+RUN --mount=type=cache,target=/root/.ccache/ \
+    # Enable ccache
+    PATH=/usr/lib/ccache:$PATH && \
+    # Fetch required upstream sources for building
+    vcs import src < /root/ws_moveit/src/moveit2_tutorials/moveit2_tutorials.repos && \
+    . "/opt/ros/${ROS_DISTRO}/setup.sh" &&\
+     . "install/setup.sh" &&\
```

Comment:
These are redundant too, because they are part of the `source` base image as well:
https://github.com/ros-planning/moveit2/blob/2de48a7aafacd45d4f9d4c667ff7337b1da31ddd/.docker/source/Dockerfile#L34